src/integrationTest/java/org/opensearch/security/SystemIndexTests.java

```diff
@@ -80,4 +80,29 @@ public void adminShouldNotBeAbleToDeleteSecurityIndex() {
             assertThat(response4.getStatusCode(), equalTo(RestStatus.FORBIDDEN.getStatus()));
         }
     }
+
+    @Test
+    public void adminShouldNotBeAbleToReadSecurityIndex() {
```

Comment:
```suggestion
    public void adminShouldNotBeAbleToReadSystemIndex() {
```

Reply:
Done in: [7459bc2](https://github.com/opensearch-project/security/pull/4774/commits/7459bc211e0281c9cd1266347b4c755915606e2e)